### DISCUSSION TITLE
Unable to replace certain windows files

### DISCUSSION THREAD

**Discussion by austinjklim** [bbmaster123/10SM#19] [General]
First off, really appreciate that you're releasing this.

I'm having this issue where i couldn't replace files such as "AppResolver" & "Windows.UI.Shellcommon.pri". 

System 32 error was there was a file/folder was in used, the pri error was some error code where a user-mapped action open.

I'm not sure i'm doing it right, I did take permission of the files. 

Win 11 22000.778 build.

**Comment by bbmaster123** [comment]
Hi friend, apologies I did not see this. I've been away from the project for a while.
Sometimes this happens due to the way those 2 files are loaded by the OS. To get around this, you need to rename each file to end in .bak or something similar, which it will allow, then move the new files into that directory. Next time you sign out/in, it will load the new files instead, allowing you to move/delete the originals.

hope that helps
cheers

**Reply by SamuraisEpic** [reply]
A better method I found was to do this all in winre so you don't have to deal with startmenuexperiencehost, explorer, or ownership at all. Just open a cmd in winre, open a notepad, then CtrlO. The open dialogue should provide a GUI file manager open with system rights where you can replace the files effortlessly
